ios/RNInstabug/ArgsRegistry.m

```diff
@@ -158,24 +158,33 @@ + (ArgsDictionary *) locales {
     return @{
         @"localeArabic": @(IBGLocaleArabic),
         @"localeAzerbaijani": @(IBGLocaleAzerbaijani),
+        @"localeBulgarian": @(IBGLocaleBulgarian),
         @"localeChineseSimplified": @(IBGLocaleChineseSimplified),
         @"localeChineseTraditional": @(IBGLocaleChineseTraditional),
+        @"localeCroatian": @(IBGLocaleCroatian),
         @"localeCzech": @(IBGLocaleCzech),
         @"localeDanish": @(IBGLocaleDanish),
         @"localeDutch": @(IBGLocaleDutch),
         @"localeEnglish": @(IBGLocaleEnglish),
+        @"localeEstonian": @(IBGLocaleEstonian),
         @"localeFrench": @(IBGLocaleFrench),
         @"localeGerman": @(IBGLocaleGerman),
+        @"localeGreek": @(IBGLocaleGreek),
         @"localeItalian": @(IBGLocaleItalian),
         @"localeJapanese": @(IBGLocaleJapanese),
         @"localeKorean": @(IBGLocaleKorean),
+        @"localeLatvian": @(IBGLocaleLatvian),
+        @"localeLithuanian": @(IBGLocaleLithuanian),
         @"localePolish": @(IBGLocalePolish),
         @"localePortugueseBrazil": @(IBGLocalePortugueseBrazil),
         @"localeRomanian": @(IBGLocaleRomanian),
         @"localeRussian": @(IBGLocaleRussian),
+        @"localeSerbian": @(IBGLocaleSerbian),
         @"localeSpanish": @(IBGLocaleSpanish),
+        @"localeSlovenian": @(IBGLocaleSlovenian),
```

Comment:
minor change: let's stick to the alphabetical order even for subsequent letters.